provider/etcd/etcd_client.go

```diff
@@ -362,3 +368,17 @@ func UnmarshalAlert(alertStr string) (*types.Alert, error) {
 	}
 	return &alert, nil
 }
+
+func getDurationFromEnv(envVar string, defaultVal int) time.Duration {
+	envVal, exists := os.LookupEnv(envVar)
+	if exists {
+		intVal, err := strconv.Atoi(envVal)
+		if err != nil {
+			return time.Duration(defaultVal) * time.Millisecond
```

Comment:
MAY write some warning messages, it helps people correct their mistakes.

Reply:
I was thinking about that too, but this happens before we have access to the logger object attached to the alerts. I'm sure there's a way, but I'm new to golang, so it wasn't obvious to me